src/wp-admin/css/edit.css

```
@@ -736,6 +736,12 @@ form#tags-filter {
 	margin-top: 5px;
 }
 
+/* Additional margin when there is a PHP notice */
+.privacy-settings.php-error #wpbody-content,
+.site-health.php-error #wpbody-content {
+	margin-top: 2em;
+}
```

Maybe we should add a margin on all pages to keep it consistent across all admin pages? What do you think?
```suggestion
/* Additional margin when there is a PHP notice */
.php-error #wpbody-content {
	margin-top: 2em;
}
```